whelk-core/src/main/groovy/whelk/Document.groovy

```diff
@@ -809,19 +808,24 @@ class Document {
             return node.booleanValue() ? depth : term
         else if (node instanceof Integer)
             return node.intValue() * depth
+        else if (node instanceof Long)
+            return node.longValue() * depth
         else if (node instanceof Map) {
             for (String key : node.keySet()) {
-                if ( !key.equals(JsonLd.MODIFIED_KEY) && !key.equals(JsonLd.CREATED_KEY)) {
+                if (key != JsonLd.MODIFIED_KEY && key != JsonLd.CREATED_KEY && key != JsonLd.RECORD_STATUS_KEY) {
```

Comment:
Are we 100% about exempting recordStatus from checksum? It is true that it changes when modified is different from created; but can it never change in any other case? (Apart from Deleted, which I presume is practically irrelevant here...?)